src/template.rs

```diff
@@ -10,52 +14,81 @@ lazy_static! {
 
 #[derive(Debug, Clone)]
 pub struct Template {
-    src: Bytes,
-    dynamic: bool,
+    src: String,
+    src_bytes: Bytes,
```

Comment:
I wonder if we can move these into an enum since from what I can tell `src_bytes` is only used when `has_fields` and `has_ts` is false?

Reply:
Maybe. The idea here was that we'd do something more optimal in `render_string` vs `render` so that any template could do either with a minimum of copying data around. That may or may not be useful.